### PR TITLE
CI: change condition for ManageArtifacts stage

### PR DESCRIPTION
**Pull request by AAndrisa**
After default branch changed to main this stage was omitted because of the condition.

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -18,7 +20,7 @@ pr:
   branches:
     include:
     - main
-    - master
+    - next_stable
```
Comment (pcercuei) [inline azure-pipelines.yml:23]: Unrelated change?

**Comment by AAndrisa** [comment]
V2: made two separate commits

**Review by SRaus — APPROVED** [review]

**Review by pcercuei — APPROVED** [review]
